README.md

```diff
@@ -39,6 +39,22 @@ This project uses [`jest` 23.x](https://jestjs.io/docs/en/23.x/getting-started.h
 
 In order to add new tests for a component (e.g. `YourComponent.js`), add a `YourComponent.test.js` file to the same folder as the component you would like to test.
 
+### Lodash
+
+This project uses [`lodash`](https://lodash.com/docs/4.17) to provide some additional JavaScript helper functions. Check out lodash's developer documentation, it's super helpful.
+
+If you find a function you'd like to use (e.g. [`map`](https://lodash.com/docs/4.17.11#map)), you should import it individually at the top of your file like so:
+
+```javascript
+import map from 'lodash/map';
+```
+
+### Testing
+
+This project uses [`jest` 23.x](https://jestjs.io/docs/en/23.x/getting-started.html) for unit testing. Use the `npm test` command from within the `frontend` folder to run the existing tests.
+
+In order to add new tests for a component (e.g. `YourComponent.js`), add a `YourComponent.test.js` file to the same folder as the component you would like to test.
+
```

Comment:
This can be removed. It looks like it is being duplicated from above for some reason.